#CREATE request parameters
string[] sentence
--------------------------------
#CREATE response parameters
bool speaking